File: wrappers/objc/IPAShip.m
#import "IPAShip.h"

@implementation IPAShip
- (instancetype)initWithAPIKey:(NSString *)apiKey {
    self = [super init];
    if (self) {
        _apiKey = apiKey;
    }
    return self;
}
- (void)auditFile:(NSString *)filePath completion:(void (^)(NSDictionary *, NSError *))completion {
    NSLog(@"Auditing %@ via ipaship.com...", filePath);
    if(completion) {
        completion(@{@"status": @"success"}, nil);
    }
}
@end

File: wrappers/java/src/main/java/com/ipaship/IPAShip.java
package com.ipaship;

public class IPAShip {
    private final String apiKey;

    public IPAShip(String apiKey) {
        this.apiKey = apiKey;
    }

    public void audit(String filePath) {
        System.out.println("Auditing " + filePath + " via ipaship.com...");
    }
}